sql/updates/latest-dev.sql

```diff
@@ -46,3 +46,23 @@ BEGIN
     WHERE mat_hypertable_id OPERATOR(pg_catalog.=) _cagg_data.mat_hypertable_id;
 END;
 $BODY$;
+CREATE TABLE _timescaledb_internal.job_errors (
+  job_id integer not null, 
+  pid bigint, -- is int enough? what's the max pid on a system
+  start_time timestamptz,
+  finish_time timestamptz,
+  error_data jsonb
+);
+
+SELECT pg_catalog.pg_extension_config_dump('_timescaledb_internal.job_errors', '');
+
+CREATE OR REPLACE VIEW timescaledb_information.job_errors AS 
+SELECT job_id, pid, start_time, finish_time, error_data->'sqlerrcode' as sqlerrcode, 
+(
+  CASE WHEN error_data IS NOT NULL THEN
+    (error_data->'message')::text
+  ELSE
+    'job crash detected, see server logs'
+  END) AS err_message, error_data->'detail' as err_detail, 
+error_data->'hint' as err_hint FROM _timescaledb_internal.job_errors;
```

Comment:
```suggestion
SELECT
    job_id,
    pid,
    start_time,
    finish_time,
    error_data ->> 'sqlerrcode' AS sqlerrcode,
    COALESCE(error_data ->> 'message', 'job crash detected, see server logs') AS err_message,
    error_data ->> 'detail' AS err_detail,
    error_data ->> 'hint' AS err_hint
FROM
    _timescaledb_internal.job_errors;
```

Note the usage of `COALESCE` and the operator `->>` that result in a TEXT type instead of JSONB

Reply:
Actually I think all these fields should be `TEXT` instead of `JSONB` and I'll change them accordingly.